src/lib/menu/menu-interface.ts

```diff
@@ -0,0 +1,11 @@
+import {EventEmitter, TemplateRef} from '@angular/core';
+import {MenuPositionX, MenuPositionY} from './menu-positions';
+
+export interface MdMenuInterface {
```

Comment:
Let's call this `MdMenuPanel`
(generally trying to avoid using the word "interface" in the names of interfaces)